### ISSUE TITLE
Přidat mapillary street-view

### ISSUE THREAD

**Issue by zbycz** [osmcz/osmcz#16]
Podobně jako google street view možno vložit overlay fotek z http://mapillary.com a prohlížeč. 

Pár nápadů je zde: https://www.mapillary.com/integrations.html

**Comment by vop-cz** [comment]
Ano, to prilaka spoustu novych tvurcu - porizovatelu streetview fotografii...

**Comment by Krakonos** [comment]
Tohle asi dá hodně práce, ale co mě zajímá mě, jestli jde mapillary použít opravdu jako streeview? Konkrétně:

Mapillary je sequence-based, takže si člověk dříve nebo později musí zvolit sekvenci od jednoho člověka, která danou ulicí prochází, a tu prohlížet. Nedávno umožnili také podívat se bokem, pokud tma takové obrázky jsou; umí ale něco jako "vyber nejkvalitnější track a ten propaguj uživateli, dokud si někdo neřekne o všechny dostupné". Jde mi o to,že třeba na Karlově mostě je toho hodně, ale vybrat dobrej track není úplně jednoduché, což je hodně škoda a navíc se to bude zhoršovat s přibývajícími mapillalery.

**Comment by zbycz** [comment]
To je dobrý postřeh. No nic lepšího než co je aktuálně na mapillary.com asi nevytvoříme. Tj. pokud vím je možné buď se posouvat po náhodných obrázcích, nebo se tam tuším něco zaškrtne a sleduju aktuální sekvenci. 

Až je těch fotek víc, začne to dělat plynulé přechody - [viz jejich blog](http://blog.mapillary.com/update/2015/07/30/better-transitions.html)
